packages/fxa-payments-server/src/lib/provider.ts

```diff
@@ -0,0 +1,9 @@
+function isStripe(provider: string) {
+  return provider === 'stripe';
+}
+
+function isPaypal(provider: string) {
+  return provider === 'paypal';
+}
+
+export { isStripe, isPaypal };
```

Comment:
I just started on something similar for #7098, where I created a lib/customer.ts.  One thing I noticed is that the type on Customer.payment_provider is just a string or undefined, which we can narrow down to the values returned by the server.  I have: 

```
export const PaymentProvider = {
  stripe: 'stripe',
  paypal: 'paypal',
  none: 'not_chosen',
} as const;

export type PaymentProvider = typeof PaymentProvider;
```

with `payment_provider?: PaymentProvider[keyof PaymentProvider];` in the Customer type.

Don't know if you want to include something like that in this PR, or I can follow up in mine, but, long story short, we can replace those string literals in the comparisons with `PaymetProvider.stripe` and `PaymentProvider.paypal`.